apps/base/arkham/pihole.yaml

```diff
@@ -0,0 +1,116 @@
+apiVersion: helm.toolkit.fluxcd.io/v2
+kind: HelmRelease
+metadata:
+  name: pihole
+  namespace: arkham
+spec:
+  releaseName: pihole
+  chart:
+    spec:
+      chart: onechart
+      sourceRef:
+        kind: HelmRepository
+        name: onechart
+        namespace: flux-system
+  interval: 12h
+  values:
+    image:
+      repository: pihole/pihole # {"$imagepolicy": "flux-system:pihole-policy:name"}
+      tag: "2025.11.0" # {"$imagepolicy": "flux-system:pihole-policy:tag"}
+    resources:
+      requests:
+        cpu: "100m"
+        memory: "256Mi"
+      limits:
+        cpu: "1"
+        memory: "1Gi"
+    vars:
+      TZ: ${TZ:=Etc/UTC}
+      WEBPASSWORD: ${PIHOLE_PASSWORD:=admin}
+      FTLCONF_LOCAL_IPV4: ${PIHOLE_IP:=0.0.0.0}
```

Comment:
configuration key IpV4 in the section Local doesn't exist
Check
- https://docs.pi-hole.net/docker/configuration/#environment-variables
- https://docs.pi-hole.net/ftldns/configfile/
Remove it